components/authz-service/storage/postgres/migration/sql/77_update_infra_service_policies.up.sql

```diff
@@ -0,0 +1,15 @@
+
+-- append "infra:infraServers:list" & "infra:infraServers:get" to the editor role's actions
+UPDATE iam_roles
+    SET actions = '{infra:infraServers:list,infra:infraServers:get}' || actions
+    WHERE
+        id = 'editor' AND
+        NOT actions @> '{infra:infraServers:list,infra:infraServers:get}';
+
+
+-- append "infra:infraServers:list" & "infra:infraServers:get" to the viewer role's actions
+UPDATE iam_roles
+    SET actions = '{infra:infraServers:list,infra:infraServers:get}' || actions
+    WHERE
+        id = 'viewer' AND
+        NOT actions @> '{infra:infraServers:list,infra:infraServers:get}';
```

Comment:
Could be more concise without loss of clarity:
```suggestion
UPDATE iam_roles
    SET actions = '{infra:infraServers:list,infra:infraServers:get}' || actions
    WHERE
        (id='editor' or id='viewer') AND
        NOT actions @> '{infra:infraServers:list,infra:infraServers:get}';
```